### ISSUE TITLE
W0511: Doesn't detect TODO in docstrings

### ISSUE THREAD

**Issue by mueller-ma** [pylint-dev/pylint#9255]
### Bug description

```python
# TODO: Write good doc strings.

def foobar():
    """
    TODO: Implement
    """
```

### Command used

```shell
pylint todo.py
```

### Pylint output

```python
************* Module todo
todo.py:1:1: W0511: TODO: Write good doc strings. (fixme)
todo.py:1:0: C0114: Missing module docstring (missing-module-docstring)

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

### Expected behavior

I expect the `TODO: Implement` to lead to an error reported by pylint.

### Pylint version

```shell
pylint 3.0.2
astroid 3.0.1
Python 3.11.2 (main, Mar 13 2023, 12:18:29) [GCC 12.2.0]
```

### OS / Environment

Debian 12 in WSL 2

**Comment by Pierre-Sassoulas** [comment]
Thank you @shenjunjie2003, I assigned you to it :)

**Comment by shenjunjie2003** [comment]
When I am using dev pylint, the fixme words in comments are not being detected. But they are indeed being detected in the version 3.0.2 which I am currently using.

**Comment by Pierre-Sassoulas** [comment]
Might be due to pylint's pylintrc influencing what you do. I would start by creating a functional test in tests/functional with an associated conf file enabling fixme.
